Source/SharpDX.Direct3D12/GraphicsCommandList.cs

```diff
@@ -298,15 +290,49 @@ public void SetScissorRectangles(params SharpDX.Mathematics.Interop.RawRectangle
         /// <summary>	
         /// Binds an array of scissor rectangles to the rasterizer stage. 
         /// </summary>	
-        /// <param name="numRects">No documentation.</param>	
-        /// <param name="rectsRef">No documentation.</param>	
-        /// <include file='.\..\Documentation\CodeComments.xml' path="/comments/comment[@id='ID3D12GraphicsCommandList::RSSetScissorRects']/*"/>	
+        /// <param name="rectangle">No documentation.</param>	
         /// <msdn-id>dn903899</msdn-id>	
         /// <unmanaged>void ID3D12GraphicsCommandList::RSSetScissorRects([In] unsigned int NumRects,[In, Buffer] const RECT* pRects)</unmanaged>	
         /// <unmanaged-short>ID3D12GraphicsCommandList::RSSetScissorRects</unmanaged-short>	
         public unsafe void SetScissorRectangles(SharpDX.Mathematics.Interop.RawRectangle rectangle)
         {
             SetScissorRectangles(1, new IntPtr(&rectangle));
         }
+
+        public void BeginEvent(string name)
```

Comment:
Sorry, I missed that part, could you copy the generated comments to BeginEvent and SetMarker